tests/messages_tests/test_models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
```

Comment:
This file should be `models.py` -- did you try running the tests locally? There are ImportErrors with `$ ./tests/runtests.py messages_tests`